### DISCUSSION TITLE
Add `emailRedirectTo` option to other email confirmation commands

### DISCUSSION THREAD

**Discussion by bombillazo** [supabase/supabase#14884] [Feature Requests]
Currently, only the `signUp` and `resetPasword` auth commands in the client support specifying the `emailRedirectUrl` and `redirectUrl` (respectively) to control where the user is directed after clicking on the confirmation URL in the email.

For better UX and control, we would also like to add a redirect URL to the `resend` command which is used to confirm other email events.

**Comment by mtte** [comment]
This already works, I checked the resend implementation in GoTrue. The only limiting factor for now is that gotrue-js does not allow to specify the redirect_to param. 

Manually sending a request with the redirect_to works:
`POST <GoTrue-URL>/resend?redirect_to=<your-redirect-url>`

**Reply by mtte** [reply]
would be fixed with https://github.com/supabase/gotrue-js/pull/719
